src/App.js

```diff
@@ -14,6 +14,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Books />} />
```

Comment:
- [ ] When I opened app, the route is ` /` it is supposed to show books, but I can see an empty page.